.github/workflows/code-testing.yml

```diff
@@ -119,6 +119,28 @@ jobs:
         run: pip install tox tox-gh-actions
       - name: "Run pytest via tox for ${{ matrix.python }}"
         run: tox
+  test-performance:
+    name: Codespeed for all supported python versions
+    runs-on: ubuntu-20.04
+    needs: [lint-python, type-python]
+    strategy:
+      matrix:
+        python: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
Hey @gmuloc, thanks for using CodSpeed!

For the tool to work, you will need to have a single upload to CodSpeed per pull request.
So here I would recommend not running the job in a matrix, and instead run it once using the latest Python version: 3.12